src/main/kotlin/org/ozwillo/dcimporter/web/Routes.kt

```diff
@@ -43,6 +43,11 @@ class Routes(private val statusHandler: StatusHandler,
                     POST("/{siret}/consultation/{reference}/piece", marchePublicHandler::createPiece)
                     PUT("/{siret}/consultation/{reference}/piece/{uuid}", marchePublicHandler::updatePiece)
                     DELETE("/{siret}/consultation/{reference}/piece/{uuid}", marchePublicHandler::deletePiece)
+
+                    //Registres
+                    POST("/{siret}/registre/type/{type}/{reference}", marchePublicHandler::refreshDatacoreRegistreForGivenConsultation)
```

Comment:
I don't understand this endpoint.

Reply:
This endpoint have been removed from last version